sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -223,6 +223,31 @@ export interface PeekMessagesOptions extends OperationOptionsBase {
     fromSequenceNumber?: Long;
 }
 
+// @public
+export interface ProcessErrorContext {
+    entityPath: string;
+    errorSource: "complete"
```

Comment:
Why no docs for complete?

Reply:
Because  it doesnt need anything to complete it, it is complete all on its own (alright, I'll show myself out now)

Reply:
So actually the odd thing is why the other members have comments at all since this is the api.md!

I'm thinking the way I documented the string union is either incorrect or perhaps can't be doc'd the way that it is because it appears to have confused the tool (and the generated type definition). I'll take a look.

Reply:
Yeah, I saw that.. it is interesting.

Reply:
Moved to just being a larger jsdoc comment on the member itself.